### PR TITLE
`use-deliver`: Review contents of exams section

### PR DESCRIPTION
**Pull request by razvand**
Review contents of "Exams" section.

### CODE REVIEW

File: chapters/use-deliver/exams/reading/README.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ Reviewer: Liza Babu
...
 Exams aim to measure comprehension, critical thinking, and problem-solving skills.
 
+RD: Mention they use items in the [drills](../../../develop-organize/drills/reading/README.md).
+The educator selects items in drills are creates the exam "paper" from those.
+
+RD: Mention attributes of an ideal exam, content-wise, such as:
+
+- good content coverage
+- good difficulty coverage
+- good connection to taught class materials
```
[the remark this answers is not in the document]
Comment (razvand) [reply chapters/use-deliver/exams/reading/README.md]: gotche

File: chapters/use-deliver/exams/reading/README.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ Reviewer: Liza Babu
...
+
+RD: Mention how it would be "generated" and present to learners: selection, digital transformation, digital presentation or printing.
```
[the remark this answers is not in the document]
Comment (razvand) [reply chapters/use-deliver/exams/reading/README.md]: roger

File: chapters/use-deliver/exams/reading/README.md [hunk 3]
```diff
@@ -1,11 +1,33 @@
...
 
-Owner: Alex Apostolescu
-Reviewer: Liza Babu
-
 Exams are assessments designed to evaluate learners' understanding and application of the course material.
 Exams aim to measure comprehension, critical thinking, and problem-solving skills.
 
+Regarding content, an ideal exam must have:
+
+- good content coverage
```
Comment (razvand) [inline chapters/use-deliver/exams/reading/README.md:8]:
```suggestion
- **good content coverage**
```

File: chapters/use-deliver/exams/reading/README.md [hunk 4]
```diff
@@ -1,11 +1,33 @@
...
+
+  An ideal exam should ensure that all subjects studied have an equal chance of being included.
+  It should essentially mirror the content repository.
+  For example, theoretical components might be drawn from sources like [drills] or [lectures], while practical, hands-on elements, could be inspired by [assignments] and [guides].
+
+- good difficulty coverage
```
Comment (razvand) [inline chapters/use-deliver/exams/reading/README.md:14]:
```suggestion
- **good difficulty coverage**
```

File: chapters/use-deliver/exams/reading/README.md [hunk 5]
```diff
@@ -1,11 +1,33 @@
...
 Exams are assessments designed to evaluate learners' understanding and application of the course material.
 Exams aim to measure comprehension, critical thinking, and problem-solving skills.
 
+Regarding content, an ideal exam must have:
+
+- good content coverage
+
+  An ideal exam should ensure that all subjects studied have an equal chance of being included.
+  It should essentially mirror the content repository.
+  For example, theoretical components might be drawn from sources like [drills] or [lectures], while practical, hands-on elements, could be inspired by [assignments] and [guides].
```
Comment (razvand) [inline chapters/use-deliver/exams/reading/README.md:12]: Add proper links.

**Review by razvand — COMMENTED** [review]
Rename `Exams` to `Tests` throughout the section. I think it look good overall.

**Comment by Alex-deVis** [comment]
@razvand, this is ready for merge, waiting on @lizababu's approval.

**Review by lizababu — APPROVED** [review]

**Pull request merged**